src/ol/webgl/PostProcessingPass.js

```diff
@@ -265,7 +268,23 @@ class WebGLPostProcessingPass {
 
     // render the frame buffer to the canvas
     gl.enable(gl.BLEND);
-    gl.blendFunc(gl.ONE, gl.ONE_MINUS_SRC_ALPHA);
+    if (glBlendEquation !== null && glBlendEquation !== undefined) {
+      gl.blendEquation(glBlendEquation);
+    } else {
+      gl.blendEquation(gl.FUNC_ADD);
+    }
+
+    if (
+      glBlendFuncSRC !== null &&
+      glBlendFuncSRC !== null &&
+      glBlendFuncSRC !== undefined &&
+      glBlendFuncSRC !== undefined
+    ) {
+      gl.blendFunc(glBlendFuncSRC, glBlendFuncDST);
```

Comment:
I think this is supposed to check `glBlendFuncDST` instead of only `glBlendFuncSRC`

Reply:
oh yes, definitely!